components/Card.vue

```diff
@@ -1,28 +1,38 @@
 <template>
   <NuxtLink
-    class="flex flex-col items-center bg-white border border-gray-200 rounded-lg shadow md:flex-row hover:bg-gray-100 dark:border-gray-700 dark:bg-gray-800 dark:hover:bg-gray-700"
+    class="flex flex-col h-auto md:px-10 px-6 py-6 rounded-2xl hover:bg-stone-200 md:border-2 border border-stone-500 hover:border-stone-700"
     :to="props.path"
   >
-    <img
-      class="object-cover w-full rounded-t-lg h-96 md:h-auto md:w-48 md:rounded-none md:rounded-l-lg"
-      :src="props.image ? `thumbnails/${props.image}` : 'wed-logo.png'"
-      :alt="props.title"
-    />
-    <div class="p-4 w-full leading-normal">
+    <div>
+      <p class="text-sm font-medium text-stone-500">
+        <time :datetime="$dayjs(props.date).format('YYYY-MM-DD')">{{
```

Comment:
```suggestion
        <time :datetime="$dayjs(props.date).format('YYYY-MM-DD')">
          {{$dayjs(props.date).format('LL')}}
        </time>
```

細かいんですけど、こっちの方が良さそうですね

Reply:
細かい部分のチェックありがとうございます😭
フォーマッタに直されなかったのでどっちでも問題ないみたいですが、他のコードと書き方合わせるためにも {{ }} で一行使う書き方に修正しました！